.eslintrc.cjs

```diff
@@ -26,8 +26,9 @@ module.exports = {
         },
     ],
     rules: {
+        'require-await': 'error',
```

Comment:
Noticed some places where this rule would help catch bad `async` function signatures